## Q9) Calculate Skewness, Kurtosis & draw inferences on the following data
   ### SP and Weight(WT)
### Use Q9_b.csv


In [1]:
import pandas as pd
import warnings
warnings.filterwarnings('ignore')

In [3]:
data=pd.read_csv('Q9_b.csv')

In [4]:
data.head()

Unnamed: 0          SP         WT
0           1  104.185353  28.762059
1           2  105.461264  30.466833
2           3  105.461264  30.193597
3           4  113.461264  30.632114
4           5  104.461264  29.889149

# Skewness:

In [7]:
data.SP.skew()

1.6114501961773586

  **The skewness for SP is 1.611. This means that the skewness is not symmetrical around mean and it is not noramally distributed.**

In [8]:
data.WT.skew()

-0.6147533255357768

**The skewness for WT is -0.6147. This means that the skewness is not symmetrical around mean and it is not noramally distributed.**

# Kurtosis:

In [6]:
data.SP.kurtosis()

2.9773289437871835

**The kurtosis for SP is 2.97,this means that there are less outliers.**  

In [11]:
data.WT.kurtosis()

0.9502914910300326

**The kurtosis for WT is 0.9502,this means that there are less outliers.**  